src/vaadin-grid-pro-edit-column.html

```diff
@@ -206,10 +206,14 @@
 
       _getEditorComponent(cell) {
         return this.editorType === 'custom' ?
-          cell._content.firstElementChild :
+          this._getCustomEditorComponent(cell) :
           cell._content.querySelector(this._getEditorTagName(cell));
       }
 
+      _getCustomEditorComponent(cell) {
+        return Array.from(cell._content.querySelectorAll('*')).filter(node => node.validate || node.checkValidity)[0];
```

Comment:
Seems like we are introducing a new constraint here? If so, it should be documented and explained.

Reply:
Updated.